common/js/src/popup-window/popup-opener.js

```diff
@@ -62,6 +62,8 @@ const DEFAULT_DIALOG_CREATE_WINDOW_OPTIONS = {
   'visibleOnAllWorkspaces': true
 };
 
+const dialogWindowWidth = 500;
```

Comment:
I don't think this constant is needed - first, because the callers provide the "width" parameter, and, second, it's conceptually wrong to hardcode the width in this common file.

Reply:
This was a workaround because the default size is bigger than that in the APP mode.

Reply:
The root cause is here I think:
https://github.com/GoogleChromeLabs/chromeos_smart_card_connector/blob/master/third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/user-prompting-checker.js#L55

(We need to pass `'width'` without nesting inside `'innerBounds'`)

Reply:
After fixing that place, the `dialogWindowWidth` constant could be removed I believe.